h/static/scripts/directive/annotation.js

```diff
@@ -282,6 +283,10 @@ function AnnotationController(
      */
     vm.timestamp = null;
 
+    // keeps track of whether the timer that periodically refreshes
+    // vm.timestamp is active
+    timestampUpdateScheduled = false;
```

Comment:
Nitpicky, but in this file I've gone for jsdoc '/**' comments for the docstrings of methods/function/variables, and I've used capital letters and full stops!